web_decimal_numpad_dot/static/src/js/numpad_dot.js

```diff
@@ -0,0 +1,45 @@
+/* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_decimal_numpad_dot.FieldFloat', function (require) {
+"use strict";
+
+var core = require('web.core');
+var formats = require('web.formats');
+var form_widgets = require('web.form_widgets');
+var translation = require('web.translation');
+var l10n = translation._t.database.parameters;
+
+var FieldFloatDecimalDot = form_widgets.FieldFloat.extend({
+
+    events: {
+        'keypress': function (e) {
+            if (e.which === 46 || e.which === 44) {
+                e.preventDefault();
+                var input = this.$input || this.$('input');
+                var l10n = translation._t.database.parameters;
+                if (!_.str.contains(input.val(), l10n.decimal_point)) {
+                    try {
+                        var caret_pos = input[0].selectionStart;
+                        var selection_end = input[0].selectionEnd;
+                        var cur_val = input.val();
+                        var newval = cur_val.substring(0, caret_pos) + l10n.decimal_point + cur_val.substring(selection_end);
+                        input.val(newval);
+                        input[0].selectionStart = input[0].selectionEnd = caret_pos + 1;
+                    } catch (e) {
+                        //fallback to appending if no caret position can be determined
+                        input.val(input.val() + l10n.decimal_point);
+                    }
+                }
+            }
+        },
+    }
+
+});
+
+core.form_widget_registry.add('float', FieldFloatDecimalDot);
+
+return {
+    FieldFloat: FieldFloatDecimalDot,
+};
+
+});
```

Comment:
I think this line should be removed.

Reply:
No, it's OK, it's closing the `odoo.define()` function.